src/SelfAssessment/EmergencyAssessment.js

```diff
@@ -51,41 +50,20 @@ export const EmergencyAssessment = ({ navigation }) => {
 const ChoiceButtons = ({ agreePress, disagreePress }) => {
   const { t } = useTranslation()
 
-  const iAmLabel = `${t("assessment.i_am")} `
-  const iAmNotLabel = `${t("assessment.i_am_not")} `
-
   return (
     <View>
       <Button
-        textStyle={styles.choiceTextStyle}
-        buttonStyle={styles.choiceButtonsStyle}
         onPress={agreePress}
-        title={
-          <RTLEnabledText style={styles.boldText}>
-            {iAmLabel}
-            <RTLEnabledText style={styles.regularText}>
-              {t("assessment.experiencing_symptoms")}
-            </RTLEnabledText>
-          </RTLEnabledText>
-        }
-        backgroundColor={Colors.white}
-        textColor={Colors.black}
+        label={t("assessment.experiencing_symptoms")}
+        style={styles.button}
+        textStyle={styles.buttonText}
       />
-      <View style={styles.disagreeButtonContainerStyle}>
+      <View style={styles.disagreeButtonContainer}>
         <Button
-          textStyle={styles.choiceTextStyle}
-          buttonStyle={styles.choiceButtonsStyle}
           onPress={disagreePress}
-          title={
-            <RTLEnabledText style={styles.boldText}>
-              {iAmNotLabel}
-              <RTLEnabledText style={styles.regularText}>
-                {t("assessment.experiencing_any_symptoms")}
-              </RTLEnabledText>
-            </RTLEnabledText>
-          }
-          backgroundColor={Colors.white}
-          textColor={Colors.black}
+          label={t("assessment.not_experiencing_any_symptoms")}
```

Comment:
Not sure if there was a need for having the label broken apart here but visually it had the `I am` and the `I am not` in bold, maybe worth checking that this change is safe to make with product

Reply:
Great catch can you check out this commit reverting this change to use bold text?

a0f4858dcbf51fdffd37e84bbe2b505c691f88ea